src/state/slices/tradeQuoteSlice/helpers.ts

```diff
@@ -27,7 +47,7 @@ const convertCryptoBaseUnitToUsdPrecision = (
 // NOTE: "Receive side" refers to "last hop AND buy asset AND receive account".
 // TODO: we'll need a check to ensure any fees included here impact the final amount received in the
 // receive account
-const getReceiveSideAmountsCryptoBaseUnit = ({
+const _getReceiveSideAmountsCryptoBaseUnit = ({
```

Comment:
the undersacore is actually so helpful 🧠

Reply:
Yep, makes you think twice!